guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -1,20 +1,19 @@
 [id="Creating_an_Activation_Key_{context}"]
 = Creating an activation key
```

Comment:
As mentioned in https://github.com/theforeman/foreman-documentation/pull/2745#discussion_r1504337915 , should add a blurb here like

```
Create an activation key to assign various attributes to hosts at registration time.
```